Tests/SwiftLintFrameworkTests/ExtendedStringTests.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ExtendedStringTests.swift
+//  SwiftLintFrameworkTests
+//
+//  Created by Noah Gilmore on 4/5/20.
+//
```

Comment:
There are no header comments in other tests. Can we remove these?

Reply:
Thanks for the review! Sure, will take care of this soon.